### ISSUE TITLE
lighter does not visually change between opened and closed states

### ISSUE THREAD

**Issue by mircowuffwuff** [mircowuffwuff/create-diesel-generators-fabric-restitched#6]
to reproduce:

1. grab lighter
2. rightclick in the air
3. witness the sound playing but the lighters lid not opening:
![image](https://github.com/user-attachments/assets/d9e5e6b8-00f7-4ec2-b34f-c06a8f70d52a)

when it should look like this by now:
![image](https://github.com/user-attachments/assets/561e9843-2eb9-441a-8a2f-9725075ba38d)

**Comment by mircowuffwuff** [comment]
problem probably lies somewhere in LighterItem.java

**Comment by mircowuffwuff** [comment]
> problem probably lies somewhere in LighterItem.java

taking that back after looking into it more deeply. it seems to be more likely to happen due to the broken LighterItemRenderer being unused

**Comment by mircey** [comment]
changing
```java
    public static Map<String, Pair<CDGPartialModel, Pair<CDGPartialModel, CDGPartialModel>>> lighterSkinModels = new HashMap<>();
    public static void init(){}
    public static void initSkins(){
        lighterSkinModels.clear();
        lighterSkinModels.put("standard", Pair.of(new CDGPartialModel(new ResourceLocation("createdieselgenerators:item/lighter"))
                , Pair.of(new CDGPartialModel(new ResourceLocation("createdieselgenerators:item/lighter_open"))
                        , new CDGPartialModel(new ResourceLocation("createdieselgenerators:item/lighter_ignited")))));
        CreateDieselGeneratorsClient.lighterSkins.forEach((name, skinId) -> {
            lighterSkinModels.put(skinId, Pair.of(new CDGPartialModel(new ResourceLocation("createdieselgenerators:item/lighter/"+skinId))
                    , Pair.of(new CDGPartialModel(new ResourceLocation("createdieselgenerators:item/lighter/"+skinId+"_open"))
                    , new CDGPartialModel(new ResourceLocation("createdieselgenerators:item/lighter/"+skinId+"_ignited")))));
        });
    }
```
to
```java
    public static void initSkins(){
        lighterSkinModels.clear();
        lighterSkinModels.put("standard", Pair.of(new PartialModel(new ResourceLocation("createdieselgenerators:item/lighter"))
                , Pair.of(new PartialModel(new ResourceLocation("createdieselgenerators:item/lighter_open"))
                        , new PartialModel(new ResourceLocation("createdieselgenerators:item/lighter_ignited")))));
        CreateDieselGeneratorsClient.lighterSkins.forEach((name, skinId) -> {
            lighterSkinModels.put(skinId, Pair.of(new PartialModel(new ResourceLocation("createdieselgenerators:item/lighter/"+skinId))
                    , Pair.of(new PartialModel(new ResourceLocation("createdieselgenerators:item/lighter/"+skinId+"_open"))
                    , new PartialModel(new ResourceLocation("createdieselgenerators:item/lighter/"+skinId+"_ignited")))));
        });
    }
```
in [PartialModels.java](https://github.com/mircey/create-diesel-generators-fabric-restitched/blob/1.20.1/src/main/java/com/jesz/createdieselgenerators/PartialModels.java) and uncommenting
```java
          //  .transform((CreateRegistrate.customRenderedItem(() -> LighterItemRenderer::new)))
```
to
```java
            .transform((CreateRegistrate.customRenderedItem(() -> LighterItemRenderer::new)))
```
in [ItemRegistry.java](https://github.com/mircey/create-diesel-generators-fabric-restitched/blob/1.20.1/src/main/java/com/jesz/createdieselgenerators/items/ItemRegistry.java)

solves this

**Comment by mircowuffwuff** [comment]
fixed in 4d7582b, will be included in rel3+ versions of the mod
